websites/G/Goodreads/presence.ts

```diff
@@ -0,0 +1,35 @@
+const presence = new Presence({
+	clientId: "765622778317373440"
+}),
+elapsed = Math.floor(Date.now() / 1000);

```

Comment:
```suggestion
elapsed: number = Math.floor(Date.now() / 1000);
```

TypeScript *usually* knows this, but it doesn't hurt to add it.

Reply:
Is not necessary to be added

Reply:
Typescript already knows it because what is assigned is a number.